packages/embark-ui/src/components/GasStation.js

```diff
@@ -50,6 +50,10 @@ class GasStation extends Component {
   }
 
   getCurrentGas() {
+    const formattedGas = this.getGasOracleFormatted();
+    if (!formattedGas.length) {
+      return 'Too few blocks';
+    }
```

Comment:
it's a personal preference: I prefer to return only 1 type if possible
it seems that the default one is number so in that case I'd return `-1`

I find it better to handle in general

Reply:
Fixed